modules/python/types/source/types.i

```diff
@@ -47,3 +47,93 @@
 
 %template(VectorRowColInt) std::vector<types::RowCol<sys::SSize_T> >;
 %template(VectorSizeT) std::vector<size_t>;
+
+// Pickle utilities
+%pythoncode
+%{
+    import cPickle as pickle
+%}
+
+%extend types::RowCol<double>
```

Comment:
You could write a macro (see `mem.i` for an example).  Normally I detest macros in pure C++ code but give some thought to if you think it's warranted here if there aren't other options.